### ISSUE TITLE
Update running_tests.md

### ISSUE THREAD

**Issue by claptimes5** [rsim/oracle-enhanced#710]
The file was last updated in 2011, and I'm having trouble getting my environment setup to run the specs. 

Thanks!

**Comment by yahonda** [comment]
Would you update which kind of errors / trouble you are getting?

**Comment by claptimes5** [comment]
Out of the box:

```
Bundler could not find compatible versions for gem "arel":
  In snapshot (Gemfile.lock):
    arel (= 6.0.3)

  In Gemfile:
    activerecord (>= 0) ruby depends on
      arel (= 7.0.0.alpha) ruby

    arel (>= 0) ruby

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

Which seems to be happening because the Gemfile is locked at Arel 6 and edge rails requires 7.

**Comment by yahonda** [comment]
If you are using `master` branch of Oracle enhanced adapter, it supports Rails 4.2. 
Would you try to see if it works by adding `, branch: '4-2-stable'` for these lines at Gemfile.

```ruby
  gem 'activerecord',   github: 'rails/rails', branch: '4-2-stable'
  gem 'activemodel',    github: 'rails/rails', branch: '4-2-stable'
  gem 'activesupport',  github: 'rails/rails', branch: '4-2-stable'
  gem 'actionpack',     github: 'rails/rails', branch: '4-2-stable'
  gem 'railties',       github: 'rails/rails', branch: '4-2-stable'
```

**Comment by scottschulthess** [comment]
+1, I'm also getting this error when bundling on master

**Comment by scottschulthess** [comment]
It's fixed by pinning to 4.2-stable

**Comment by yahonda** [comment]
Close as #714 has been merged to master. Thanks for reporting.
